internal/configs/test_file.go

```diff
@@ -839,6 +846,7 @@ var testRunOptionsBlockSchema = &hcl.BodySchema{
 		{Name: "refresh"},
 		{Name: "replace"},
 		{Name: "target"},
+		{Name: "state_alias"},
```

Comment:
Ok i'll wait for a definitive decision on your side and i'll apply the change

I agree that it may be confusing. I was also thinking about `run_key` as mentioning the state seems weird as is never appears with terraform test